com.chenyifaer.FaFaRunner.yml

```diff
@@ -0,0 +1,56 @@
+id: com.chenyifaer.FaFaRunner
+
+# https://docs.flatpak.org/en/latest/available-runtimes.html
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: fafarunner
+
+# These are the permissions that the application needs
+# Read more about finishing here: https://docs.flatpak.org/en/latest/manifests.html#finishing
+# https://docs.flatpak.org/en/latest/sandbox-permissions.html
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=dri
+  - --share=network
+
+modules:
+  - name: fafarunner
+    buildsystem: simple
+    build-commands:
+      - bsdtar -Oxf fafarunner.deb data.tar.zst | bsdtar -xf -
+      - install -Dm755 usr/share/fafarunner/fafarunner /app/bin/fafarunner
+      - cp -r usr/share/fafarunner/data usr/share/fafarunner/lib /app/bin/
+      - sed -i -e '/^Version=/d' -e 's|Icon=fafarunner|Icon=com.chenyifaer.FaFaRunner|' usr/share/applications/fafarunner.desktop
```

Comment:
don't use sed. Use the command `desktop-file-edit`.

Reply:
Which package contains the command `desktop-file-edit`?

Reply:
desktop-file-utils on your distro, but it's in the runtime by default. You don't need to add it, it works out of the box

Reply:
Maybe this?

```
desktop-file-edit --remove-key='Version' --set-icon='com.chenyifaer.FaFaRunner' "usr/share/applications/fafarunner.desktop"
```